plugins/bf1/bf1.cpp

```diff
@@ -17,74 +18,96 @@ static int fetch(float *avatar_pos, float *avatar_front, float *avatar_top, floa
 	procptr_t server_name_offset_0 = peekProcPtr(server_name_base + 0x738);
-	if (!server_name_offset_0) return false;
+	if (!server_name_offset_0)
+		return false;
 	procptr_t server_name_offset_1 = peekProcPtr(server_name_offset_0 + 0x370);
-	if (!server_name_offset_1) return false;
+	if (!server_name_offset_1)
+		return false;
 	procptr_t server_name_offset_2 = peekProcPtr(server_name_offset_1 + 0x10);
-	if (!server_name_offset_2) return false;
+	if (!server_name_offset_2)
+		return false;
 
 	// Team pointers
 	procptr_t team_base = peekProcPtr(pModule + 0x36CC128);
-	if (!team_base) return false;
+	if (!team_base)
+		return false;
 	procptr_t team_offset_0 = peekProcPtr(team_base + 0x20);
-	if (!team_offset_0) return false;
+	if (!team_offset_0)
+		return false;
 	procptr_t team_offset_1 = peekProcPtr(team_offset_0 + 0x648);
-	if (!team_offset_1) return false;
+	if (!team_offset_1)
+		return false;
 	procptr_t team_offset_2 = peekProcPtr(team_offset_1 + 0xF8);
-	if (!team_offset_2) return false;
+	if (!team_offset_2)
+		return false;
 
 	// Squad pointers
 	procptr_t squad_base = peekProcPtr(pModule + 0x31C3E40);
-	if (!squad_base) return false;
+	if (!squad_base)
+		return false;
 	procptr_t squad_offset_0 = peekProcPtr(squad_base + 0x30);
-	if (!squad_offset_0) return false;
+	if (!squad_offset_0)
+		return false;
 	procptr_t squad_offset_1 = peekProcPtr(squad_offset_0 + 0x578);
-	if (!squad_offset_1) return false;
+	if (!squad_offset_1)
+		return false;
 	procptr_t squad_offset_2 = peekProcPtr(squad_offset_1 + 0xC0);
-	if (!squad_offset_2) return false;
+	if (!squad_offset_2)
+		return false;
 	procptr_t squad_offset_3 = peekProcPtr(squad_offset_2 + 0x58);
-	if (!squad_offset_3) return false;
+	if (!squad_offset_3)
+		return false;
 
 	// Peekproc and assign game addresses to our containers, so we can retrieve positional data
-	ok = peekProc(state_offset_3 + 0x50, &state, 1) && // Magical state value: 1 when in-game and 0 when not spawned or playing.
-			peekProc(pModule + 0x36B1500, avatar_pos, 12) && // Avatar position values (X, Y and Z).
-			peekProc(camera_base + 0x2B0, camera_pos, 12) && // Camera position values (X, Y and Z).
-			peekProc(camera_base + 0x260, camera_front, 12) && // Avatar front vector values (X, Y and Z).
-			peekProc(camera_base + 0x250, camera_top, 12) && // Avatar top vector values (X, Y and Z).
-			peekProc(server_name_offset_2, server_name) && // Server name.
-			peekProc(team_offset_2 + 0x13, team) && // Team name.
-			peekProc(squad_offset_3 + 0x240, squad) && // Squad value: 0 (not in a squad), 1 (Apples), 2 (Butter), 3 (Charlie)... 26 (Zebra).
-			peekProc(squad_offset_3 + 0x244, squad_leader); // Squad leader value: 0 (False), 1 (True).
-
-	// This prevents the plugin from linking to the game in case something goes wrong during values retrieval from memory addresses.
+	ok = peekProc(state_offset_3 + 0x50, &state, 1)
+		 && // Magical state value: 1 when in-game and 0 when not spawned or playing.
```

Comment:
Why is it on a new line?

Reply:
Pretty sure that's due to the long comment. This should also be the reason why it has the `&&` at the end of the first line instead of the beginning of the second :thinking: 

I assume that putting these comments in their own line (above the code the are commenting) will fix both of these issues...

Reply:
I agree.